lib/unitarize_force_quda.cu

```diff
@@ -646,7 +646,7 @@ static double HOST_REUNIT_SVD_ABS_ERROR;
       void preTune() { ; }
       void postTune() { cudaMemset(fails, 0, sizeof(int)); } // reset fails counter
       
-      long long flops() const { return 4*4528*gauge.Volume(); } // FIXME: add flops counter
+      long long flops() const { return 4ll*4528*gauge.Volume(); } // FIXME: add flops counter
```

Comment:
is that fixed with the 411*4528*Volume or just better than before but still wrong? (Just wondering about the FIXME comment)